modules/object.py

```diff
@@ -122,19 +129,14 @@ def paste_object(editor, event):
 # Case 2: When a user selects a new Page or Section in the editor
 def build_object(editor, params):
     if params.type == 'text':
-
```

Comment:
i think i was the one that put this in here, but its not used and probably wont be since build_object is really only called for loading sections